cmd/ndm_daemonset/filter/vendorfilter.go

```diff
@@ -26,13 +26,18 @@ import (
 
 const (
 	vendorFilterKey = "vendor-filter"
+	// OpenEBSVendor is the vendor for iscsi disks created by OpenEBS
+	OpenEBSVendor = "OpenEBS"
```

Comment:
n_: May be useful to change the const variable name to "VendorValueOpenEBS"? 

also, check if go recommends any specific representation for const values. So that while they are being used in the code below, the reader can know if it is constant by looking at the naming convention.

Reply:
Go recommends using CamelCase for constants, as is in the standard library. 
I will rename it to `vendorValueOpenEBS`, so that it is un-exported .  We need to use it only in this package.

Reply:
Again, a thought why should we have the word `value` in it. 
It sounds like naming a literal `NameString`, `AddressHashMap`,  which is not recommended.

Reply:
Done.